### ISSUE TITLE
Identify/Rename devices

### ISSUE THREAD

**Issue by Kreuger** [csicar/Ning#34]
Hey would it be possible for the app to better identify the devices? The only info shown is the manufacturer and it doesnt even detect a lot of mine. I would also like the ability to rename or add a nickname to each device.

**Comment by csicar** [comment]
Yes, that should be possible. A device could be identified by the MAC and IP address. 

It would also be possible to get more information using information about the open ports.

**Comment by Gitoffthelawn** [comment]
Thank you for ning.  This is probably the top feature I would like to see.  Because this information may be private, be sure to store the data such that it automatically gets deleted if ning is uninstalled.

Import/export functionality would be nice, but not a big deal if that's too much work or would add too much bulk.
